### PR TITLE
Correct references and add manual references

### PR DESCRIPTION
**Pull request by agitter**
I reviewed the HTML after #1002 was merged and found a few reference-related issues to fix.  @rando2 can you please double-check the DOIs I updated in the text?

I also fixed some bad URL references in the vaccines section.  Those failed citations slow down builds.  Some of them should work as-is now that I used `{ }` to escape special characters in the URL.  That's a new pandoc feature.

### CODE REVIEW

File: content/20.pharmaceuticals.md [hunk 1]
```diff
@@ -252,13 +252,13 @@ The potential for antiviral effects on SARS-CoV-2 were investigated _in vitro_,
...
+A similarly sized RCT suggested no effect on the duration of symptoms among 400 patients split evenly across the intervention and control arms [@doi:10.1001/jama.2021.3071], and although meta-analyses have reported both null [@doi:10.1101/2021.05.21.21257595; @doi:10.1101/2021.01.26.21250420] and beneficial [@doi:10.1093/ofid/ofab394; @doi:10.1016/j.dsx.2021.102186; @doi:10.1097/MJT.0000000000001402; @doi:10.1002/rmv.2265; @doi:10.31219/osf.io/k37ft; @doi:10.1097/MJT.0000000000001377; @doi:10.1007/s43440-021-00245-z; @doi:10.1101/2021.04.30.21256415] effects of ivermectin on COVID-19 outcomes, the certainty is likely to be low [@doi:10.1016/j.dsx.2021.102186].
+These findings are potentially biased by a small number of low-quality studies, including the preprint that has been taken down [@url:https://gidmk.medium.com/does-ivermectin-work-for-covid-19-1166126c364a], and the authors of one have issued a notice [@doi:10.1093/ofid/ofab394] that they will revise their study with the withdrawn study removed.
```
Comment (rando2) [inline content/20.pharmaceuticals.md:260]:
```suggestion
These findings are potentially biased by a small number of low-quality studies, including the preprint that has been taken down [@url:https://gidmk.medium.com/does-ivermectin-work-for-covid-19-1166126c364a], and the authors of one [@doi:10.1093/ofid/ofab358] have issued a notice [@doi:10.1093/ofid/ofab394] that they will revise their study with the withdrawn study removed.
```
The article and the notice have separate URLs -- they are similar and apparently I managed to paste the same one twice!

**Review by rando2 — APPROVED** [review]
Thank you for fixing these @agitter! They look good to me, other than that one duplication was just me pasting the same DOI twice, so I added the correct DOI back in.

File: content/20.pharmaceuticals.md (hunk 1 above)
Comment (agitter) [reply content/20.pharmaceuticals.md:260]: That makes more sense.  Thanks for the correction.

**Comment by agitter** [comment]
I'm going to review the AppVeyor artifacts before merging.  My first attempt to update the manual references added an invalid character copied from a PDF and broke reading the file.

**Comment by rando2** [comment]
> The build step has been reduced from 44 to 14 minutes

WOW!!!!!!!! Amazing news!

**Pull request merged**